helper/geojsonify.js

```diff
@@ -3,23 +3,33 @@ var GeoJSON = require('geojson'),
     extent = require('geojson-extent'),
     outputGenerator = require('./outputGenerator');
 
-function search( docs, params ){
+// Properties to be copied when details=true
+var DETAILS_PROPS = [
+  'alpha3',
+  'admin0',
+  'admin1',
+  'admin1_abbr',
+  'admin2',
+  'local_admin',
+  'locality',
+  'neighborhood',
+  'category',
+  'address'
+];
+
 
-  // emit a warning if the doc format is invalid
-  // @note: if you see this error, fix it ASAP!
-  function warning(){
-    console.error( 'error: invalid doc', __filename );
-    return false; // remove offending doc from results
-  }
+function search( docs, params ){
 
   var details = params ? params.details : {};
   details = details === true || details === 1;
 
   // flatten & expand data for geojson conversion
-  var geodata = docs.map( function( doc ){
+  var geodata = docs.map( function( doc ) {
 
     // something went very wrong
-    if( !doc ) { return warning(); }
+    if( !doc || !doc.center_point ) {
```

Comment:
nit picking.. `!doc.hasOwnProperty('center_point')` it's really not a big deal but the original code I wrote was wrong and should have performed a key check rather than a value check here.